### PR TITLE
bpo-37904: Edition on python tutorial - section 4

### PR DESCRIPTION
**Pull request by umoqnier**
A little change on first paragraph of python tutorial to be more clearly

https://bugs.python.org/issue37904

Automerge-Triggered-By: @ericvsmith

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@umoqnier

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Doc/tutorial/controlflow.rst [hunk 1]
```diff
@@ -4,8 +4,8 @@
 More Control Flow Tools
 ***********************
 
-Besides the :keyword:`while` statement just introduced, Python knows the usual
-control flow statements known from other languages, with some twists.
+Besides the :keyword:`while` statement just introduced, Python use the usual
```
Comment (ericvsmith) [inline Doc/tutorial/controlflow.rst]: "use" should be "uses".

**Review by ericvsmith — CHANGES_REQUESTED** [review]
In addition to the grammar fix, you need to sign the CLA.

**Comment by ericvsmith** [comment]
Although now that I think about it, this change is so trivial that I don't think it needs the CLA to be signed. But if you're planning on making other contributions, please consider signing it.

File: Doc/tutorial/controlflow.rst (hunk 1 above)
Comment (umoqnier) [reply Doc/tutorial/controlflow.rst]: Sure. It's fixed now

File: Doc/tutorial/controlflow.rst [hunk 2]
```diff
@@ -4,8 +4,8 @@
 More Control Flow Tools
 ***********************
 
-Besides the :keyword:`while` statement just introduced, Python knows the usual
-control flow statements known from other languages, with some twists.
+Besides the :keyword:`while` statement just introduced, Python uses the usual
```
Comment (ericvsmith) [inline Doc/tutorial/controlflow.rst]: Travis is failing because of the trailing whitespace on this line. Please remove it.

**Review by ericvsmith — CHANGES_REQUESTED** [review]

**Comment by aeros167** [comment]
@ericvsmith:
> Although now that I think about it, this change is so trivial that I don't think it needs the CLA to be signed. But if you're planning on making other contributions, please consider signing it.

I was under the impression that the CLA has transitioned from a recommendation to strict requirement for any contribution. Is this no longer true?

**Comment by tirkarthi** [comment]
3.6 is in security fixes mode. Guess we can remove the label for backport.

File: Doc/tutorial/controlflow.rst (hunk 2 above)
Comment (umoqnier) [reply Doc/tutorial/controlflow.rst]: It's already. Sorry for that, it's my first contribution

**Review by ericvsmith — APPROVED** [review]

**Pull request merged**